sdks/python/apache_beam/transforms/environments.py

```diff
@@ -138,20 +145,33 @@ def __hash__(self):
     return hash((self.__class__, self.container_image))
 
   def __repr__(self):
-    return 'DockerEnvironment(container_image=%s)' % self.container_image
+    return 'DockerEnvironment(container_image=%s,env=%s)' % (
```

Comment:
space after the comma, I think

Reply:
I was matching the prevailing style of `ProcessEnvironment` and `ExternalEnvironment`.